docs/fleet/cli.md

```diff
@@ -377,22 +393,44 @@ viam data database hostname --org-id=abc
 | `--component-name`      | filter by specified component name  |`export`, `delete`| false |
 | `--component-type`     | filter by specified component type       |`export`, `delete`| false |
 | `--component-model`   | filter by specified component model       |`export`, `delete`| false |
+| `--dataset`   | dataset to add or remove images from     |`dataset`| true |
 | `--delete-older-than-days` | number of days, 0 means all data will be deleted | `delete` | false |
-| `--start`      | ISO-8601 timestamp indicating the start of the interval       |`export`, `delete`| false |
-| `--end`      | ISO-8601 timestamp indicating the end of the interval       |`export`, `delete`| false |
+| `--start`      | ISO-8601 timestamp indicating the start of the interval       |`export`, `delete`, `dataset`| false |
+| `--end`      | ISO-8601 timestamp indicating the end of the interval       |`export`, `delete`, `dataset`| false |
+| `--file-ids` | file-ids to add or remove from a dataset       |`dataset`| true |
+| `--location-id`      | location id for the specified dataset (only accepts one location id)       |`dataset`| true |
 | `--location-ids`      | filter by specified location id (accepts comma-separated list)       |`export`, `delete`| false |
 | `--method`       | filter by specified method       |`export`, `delete`| false |
 | `--mime-types`      | filter by specified MIME type (accepts comma-separated list)       |`export`, `delete`|false |
+| `--org-id` | org ID for the database user | `database configure`, `database hostname`, `dataset` | true |
```

Comment:
location ID for the file IDs being added/removed from the specified dataset (only accepts one location id)

Don't know if need to link this to `database configure/hostname` unless you try to link any time a flag is reused!

Reply:
We do include a flag everywhere it is reused. 👍 

Otherwise I'm not sure I understand the comment: was location ID comment intended for other comment, dupe?

Reply:
Sorry, meant to update to org ID. The comment would be 'org ID for the file ids being added or removed from the specified dataset (only accepts one org id)`